defold-spine/editor/src/spineext.clj

```diff
@@ -1008,6 +1008,7 @@
    (workspace/register-resource-type workspace
                                      :ext spine-json-ext
                                      :node-type SpineSceneJson
+                                     :textual? true
```

Comment:
While it _is_ textual, I do wonder if it in this case reflects our intended reason for "Search in files".

E.g.this is a 3'rd party format, which it's not always necessary to dig into (depending on how ppl use the search-in-files).
It is not likely they'll change anything in there, and finding strings in a spine json is probably not very common.

In short, does it make sense to have another flag (e.g. "proprietary") and also have an extra checkbox in the search dialog for searching in external files as well?

I mean, same goes for binary files as well. I search though files regardless of binary/text, and I think it's reasonable that the users might was some granularity as well.

Just a thought.

Reply:
Couldn't you want to search for bone names or things like that inside these files. I was thinking you might refer to these names from script files for example, and might want to ensure you've renamed every occurrence in both scripts and spine scenes.

Reply:
Sure, but that goes for binary files as well then. (case in point: .glb)

As for searching those things, wouldn't it be relevant to search our _parsed_ data then? E.g. which bone names did we find in the file?

Reply:
Yep! And this might in fact happen soon. :)